### DISCUSSION TITLE
What kind of token I need to configure Jira in issue-tracker?

### DISCUSSION THREAD

**Discussion by fmateosb** [projectdiscovery/nuclei#3566] [Q&A]
Good afternoon, 
I'm trying to automate the Jira tickets opening and I don´t want to use my personal password. I know that you can configure a token in issue-tracker but in Jira I only can see PAT (Personla Access Token) and it isn´t work for my.

How can configure Jira with the correct token ?

Thank you

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
